.github/workflows/build_hub.yml

```diff
@@ -190,55 +102,16 @@ jobs:
     defaults:
       run:
         working-directory: prime-router
-    strategy:
-      fail-fast: false
-      matrix:
-        language: ["java"]
 
     steps:
       - name: "Check out changes"
         uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
 
-      - name: Initialize CodeQL
-        uses: github/codeql-action/init@4fddc51e4f3b5e5e9022f35c3464736cc10e1e98
-        if: github.actor != 'dependabot[bot]'
-        with:
-          languages: ${{ matrix.language }}
-
-      - name: Set up JDK 11
-        uses: actions/setup-java@cd89f46ac9d01407894225f350157564c9c7cee2
-        with:
-          java-version: "11"
-          distribution: "adopt"
-          cache: "gradle"
-
-      - uses: gradle/wrapper-validation-action@56b90f209b02bf6d1deae490e9ef18b21a389cd4
-
-      - name: Spin up build containers
-        run: docker-compose -f docker-compose.postgres.yml up -d
-
-      - name: Build Prime Router Package
-        run: ./gradlew package -x fatjar
-
-      - name: Perform Java CodeQL Analysis
-        uses: github/codeql-action/analyze@4fddc51e4f3b5e5e9022f35c3464736cc10e1e98
-        if: github.actor != 'dependabot[bot]'
-
-      - name: "Run SonarCloud Scan"
-        uses: sonarsource/sonarcloud-github-action@db501078e936e4b4c8773d1bb949ba9ddb7b6b6a
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-        if: github.actor != 'dependabot[bot]'
+      - name: Build backend
```

Comment:
The entire docker_build_test job seems unnecessary since you've moved all the steps to build-backend.